roboteam_ai/src/control/positionControllers/PosController.cpp

```diff
@@ -35,12 +35,17 @@ PosVelAngle PosController::controlWithPID(const RobotPtr &robot, PosVelAngle tar
     pidCommand.vel = calculatePIDs(robot, target);
 
     // acceleration limiter
-    double maxAcc = control::ControlUtils::calculateMaxAcceleration(pidCommand.vel, pidCommand.angle);
-    if (prevVel == 0.0) prevVel = robot->vel.length();
-    pidCommand.vel = control::ControlUtils::accelerationLimiter(pidCommand.vel, maxAcc, prevVel);
+    if (prevVel == Vector2(0.0,0.0)) {
+        prevVel = robot->vel;
+    }
+    //OLD
```

Comment:
we have git so you don't need this

Reply:
this was for testing purposes so recompiling does not take ages. We were still testing this